sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -85,12 +85,12 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSQLContext {
     }
 
     val df5 = Seq((Seq("a", null), Seq(1, 2))).toDF("k", "v")
-    intercept[RuntimeException] {
+    intercept[Exception] {
       df5.select(map_from_arrays($"k", $"v")).collect
```

Comment:
What's the concrete exception this query throws?

Reply:
@maropu I will double check and get back. But i think it was SparkException.